src/scripts/main.js

```diff
@@ -1 +1,63 @@
 'use strict';
+
+function showNotification(isResolved, promiseNumber) {
+  const message = document.createElement('div');
+
+  message.setAttribute('data-qa', 'notification');
+
+  if (isResolved) {
+    message.classList.add('success');
+
+    message.textContent = `${promiseNumber} promise was resolved`;
+  } else {
+    message.classList.add('error');
+    message.textContent = `${promiseNumber} promise was rejected`;
+  }
+
+  document.body.append(message);
+}
+
+const leftClick = new Promise((resolve) => {
+  document.addEventListener('click', () => {
+    resolve('Third promise was resolved');
```

Comment:
FYI: you don't use this value, so you could do just `resolve()` - without arguments